tl/features/cell_context_matches.py

```diff
@@ -386,7 +417,11 @@ def compute_property_scores(self, row_column_pairs: set, n_context_columns: set)
                     int_prop['column'] = col
                     int_prop['col2'] = col2
                     properties_df_list.append(int_prop)
-        properties_df = pd.concat(properties_df_list)
+        if properties_df_list:
```

Comment:
wherever possible make the if condition explicit, `if len(properties_df_list) > 0`